### DISCUSSION TITLE
Hello from both Morocco and Ivory Coast

### DISCUSSION THREAD

**Discussion by iskode** [OpenMined/courses#53] [Introduce Yourself]
Hello, I am Ismael Koné from Ivory Coast.
I've just defended my PhD thesis in Machine Learning
applied to medical image analysis in Morocco. 
I'm planning on going back soon to Ivory Coast.
Happy to be hear and learn about privacy technology.
I can feel the excitement, commitment and care of @iamtrask and 
the entire team that vibrate and flow from inside out !!!
@yemikifouly, you speak English fluently for being non native speaker. 
Awesome, keep up with the great work !

**Comment by iamtrask** [comment]
Hi Ismael! Congratulations on defending your thesis! And welcome!

**Reply by iskode** [reply]
Thank you so much Andrew.

**Comment by em-blue** [comment]
Welcome!! :-) huge congratulations on defending your thesis - hope you enjoy the course, Dr.!

**Reply by iskode** [reply]
Thank you so much dear  @em-blue!
